ca/ca_test.go

```diff
@@ -693,12 +693,6 @@ func TestInvalidCSRs(t *testing.T) {
 		// * Signature Algorithm: sha1WithRSAEncryption
 		{"RejectBadAlgorithm", "./testdata/bad_algorithm.der.csr", nil, "Issued a certificate based on a CSR with a bad signature algorithm.", berrors.BadCSR},
 
-		// CSR generated by Go:
-		// * Random RSA public key.
-		// * CN = aaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaa.com
-		// * DNSNames = [none]
-		{"RejectLongCommonName", "./testdata/long_cn.der.csr", nil, "Issued a certificate with a CN over 64 bytes.", berrors.BadCSR},
-
```

Comment:
It's somewhat unfortunate to lose this test case -- not the end of the world, obviously, but is nice to have ca_test.go tests which ensure that the CA is properly calling VerifyCSR / NamesFromCSR.

I don't think it's worth derailing this PR, so instead can I ask you to file a bug? We should refactor TestIssuePrecertficate, TestInvalidCSRs, and TestRejectValidityTooLong so that they're a single table-driven test, and that refactored test should include this temporarily-removed test case (but expecting it to succeed instead of fail).

Reply:
Opened #7702